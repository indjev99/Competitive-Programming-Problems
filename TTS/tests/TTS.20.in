100 10
-1603.88 -10000
-10000 3901.76
-10000 -4611.93
10000 -4589.85
-8075.86 10000
4873.31 10000
-9564.33 -10000
-10000 1706.45
-10000 3682.36
10000 8053.13
-2713.04 10000
10000 8171.67
1997.18 -10000
-2760.61 10000
-9050.73 -10000
-10000 4169.52
10000 2375.37
10000 -1438.03
-365.42 -10000
-10000 8839.54
-4825.29 -10000
1103.41 10000
-2814.22 10000
-10000 -7748
-10000 -128.7
-3309.77 -10000
-10000 -8896.77
3418.43 10000
-6124.84 10000
10000 -5843.28
-6505.35 -10000
9676.76 10000
-3053.86 10000
2868.34 -10000
10000 -5596.43
-6098.37 10000
6001.16 -10000
-10000 2997.92
-8389.47 -10000
-2084.88 10000
-10000 1623.73
-6304.25 10000
1612.39 10000
6309.72 -10000
10000 1688.33
-4889.02 -10000
10000 8568.19
8054.9 -10000
8435.57 -10000
6416.85 -10000
-7802.13 -10000
10000 -9790.39
10000 -1967.07
3844.56 10000
10000 -3689.06
10000 -7735.45
-10000 965.08
10000 -9908.21
-1235.48 10000
10000 -3881.27
-10000 -3286.6
-10000 9352.06
-10000 4924.57
10000 -4590.5
-4449.58 10000
-10000 7485.34
-10000 5623.42
-6873.46 -10000
10000 9483.56
-10000 -216.73
2735.56 -10000
10000 1048.72
-286.63 -10000
4599.29 10000
-5421.11 10000
2694.29 10000
-7776.01 -10000
-10000 2824.73
10000 1476.78
-10000 9097.95
-10000 3219.91
10000 -919.49
10000 9321
6987 -10000
-10000 -4500.63
10000 -7934.37
3964.13 -10000
-1012.45 10000
5581.3 10000
-7693.65 10000
522.49 -10000
-10000 9703.3
-10000 -8386.05
-10000 8189.57
10000 5909.32
10000 8636.46
10000 4855.09
6133.37 -10000
6081.85 -10000
-10000 -5865.13
-9712.87 2379.11
-7318.18 -6815
-8899.29 -4560.99
-519.912 9986.48
-395.533 9992.17
2689.18 9631.63
4230.7 9060.97
8383.97 5450.6
7636.55 -6456.24
5209.16 -8536.08
