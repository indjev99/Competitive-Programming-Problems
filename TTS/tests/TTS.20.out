10841.810000
